draft-ferrieuxhamchaoui-quic-lossbits.md

```diff
@@ -61,13 +61,13 @@ informative:
 
 --- abstract
 
-This draft adapts the general technique described in
-draft-ferrieuxhamchaoui-tsvwg-lossbits for QUIC using reserved bits in QUIC v1
-short header.  It describes a method that employs two bits to allow endpoints to
-signal packet loss in a way that can be used by network devices to measure and
-locate the source of the loss. It further describes a way to negotiate this
-functionality as an extension in QUIC v1.
+This document defines an extension to the QUIC transport protocol to allow
+endpoints to signal packet loss in a way that can be used by network devices to
+measure and locate the source of the loss.
 
+Discussion of this work is encouraged to happen on the QUIC IETF mailing list
+<quic@ietf.org> or on the GitHub repository which contains the draft:
+<https://github.com/igorlord/draft-ferrieuxhamchaoui-tsvwg-lossbits>.
```

Comment:
Why is the -tsvwg draft mentioned here instead of the -quic- one ?

Reply:
This is the git repo name...  I can rename it...

Reply:
Or maybe give a deep link, so that people don't select the wrong file:
https://github.com/igorlord/draft-ferrieuxhamchaoui-tsvwg-lossbits/blob/master/draft-ferrieuxhamchaoui-quic-lossbits.md

Reply:
A deep link does not name a repository.  The post is to encourage a discussion and PRs.

Reply:
Per a vote taken over email, renaming the repo to draft-ferrieuxhamchaoui-lossbits and resolving.